docs/site/Routes.md

```diff
@@ -10,83 +10,120 @@ summary:
 
 ## Overview
 
-A `Route` is the mapping between your API specification and an Operation (JavaScript implementation). It tells LoopBack which function to `invoke()` given an HTTP request.
+A `Route` is the mapping between your API specification and an Operation. It
```

Comment:
Maybe it's better to replace `an Operation` with `a handler function in JavaScript, such as a controller method`

Reply:
I've expanded upon the definition under `Operations` header with your example